lib/plugins/index.js

```diff
@@ -1,19 +1,18 @@
-const { pluginPingPong } = require("./ping-pong");
-const { pluginWelcomeMessage } = require("./welcome-message");
-const { pluginRandomMessages } = require("./random-messages");
-const { PLUGIN_NAME } = require("./symbols");
+const { pluginPingPong } = require('./ping-pong')
+const { pluginWelcomeMessage } = require('./welcome-message')
+const { PLUGIN_NAME } = require('./symbols')
 
-const plugins = [pluginPingPong, pluginWelcomeMessage, pluginRandomMessages];
+const plugins = [pluginPingPong, pluginWelcomeMessage]
```

Comment:
Hiányzik az új plugin :) Illetve kérlek mindet külön új sorba írd, ahogy eredetileg volt.
```suggestion
const plugins = [
    pluginPingPong, 
    pluginWelcomeMessage,
    pluginRandomMessages
]
```

Reply:
- Igen, arra én is rájöttem hogy akadni fog a "ping"-el, de egyenlőre nem foglalkoztam vele. 
- VS Code így formázza, 80 karakter alatt egy sor, de megoldjuk...
- Hiányzott a plugin igen, mert korábban írtad hogy ne írjak bele az index.js-be, de akkor ok, beledobom.

Reply:
Beleírással nem volt baj, az újraformázás zavart :D 
Standard-al formázd a kódot. `npm run lint:fix`

Reply:
Azóta feltettem local-ba a standard-ot, meg formázgatom, úgyhogy ezzel már nem lesz gond.